src/patternfly/components/MenuToggle/menu-toggle.scss

```
@@ -412,6 +413,11 @@
       }
     }
 
+    .pf-m-split-toggle-button {
+      display: flex;
+      padding-inline-start: var(--#{$menu-toggle}--m-split-button--PaddingInlineStart);
```

When this is moved into the `.#{$menu-toggle}__button` block, you can rename this to:

```suggestion
      padding-inline-start: var(--#{$menu-toggle}__button--m-text--PaddingInlineStart);
```